k--- ./Project/QMake/GUI/MediaInfoQt.pro.buildfix	2011-02-22 20:28:31.000000000 +0100
+++ ./Project/QMake/GUI/MediaInfoQt.pro	2011-05-03 21:39:26.000000000 +0200
@@ -1,17 +1,14 @@
 # -------------------------------------------------
 # Project created by QtCreator 2010-07-23T13:03:11
 # -------------------------------------------------
-TARGET = mediainfo-gui
+TARGET = mediainfo-qt
 TEMPLATE = app
 
 # QT += webkit
 QT += xml
 DEFINES += \
     _UNICODE
-LIBS += -ldl \
-    ../../../../MediaInfoLib/Project/GNU/Library/.libs/libmediainfo.a \
-    ../../../../ZenLib/Project/GNU/Library/.libs/libzen.a \
-    -lz
+LIBS += -ldl -lmediainfo -lzen -lz
 SOURCES += ../../../Source/GUI/Qt/main.cpp \
     ../../../Source/GUI/Qt/mainwindow.cpp \
     ../../../Source/Common/Core.cpp \
@@ -57,9 +54,7 @@ RESOURCES += ../../../Source/Ressource/R
 MOC_DIR = _Automated
 UI_HEADERS_DIR = _Automated
 UI_SOURCES_DIR = _Automated
-INCLUDEPATH = ../../../Source \
-    ../../../../MediaInfoLib/Source \
-    ../../../../ZenLib/Source \
+INCLUDEPATH = ../../../Source
 QMAKE_UIC += -tr \
     Tr
 target.path = /usr/bin
